site/features.md

```diff
@@ -67,16 +67,15 @@ notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notat
 
 On $HOST1:
 
-    host1$ C=$(weave run 10.2.1.1/24 -ti ubuntu)
+    host1$ docker run -e WEAVE_CIDR=10.2.1.1/24 --name a1 -ti ubuntu
 
 And $HOST2:
 
-    host2$ C=$(weave run 10.2.1.2/24 -ti ubuntu)
+    host2$ docker run -e WEAVE_CIDR=10.2.1.2/24 --name a2 -ti ubuntu
 
 Then on $HOST1...
 
-    host1$ docker attach $C
-    root@28841bd02eff:/# ping -c 1 -q 10.2.1.2
+    root@a1:/# ping -c 1 -q 10.2.1.2
```

Comment:
This IPs have actually been inserted into DNS as we specified `--name` - should I use the name here instead of the IP?